internal/smbcc/container_config.go

```diff
@@ -99,11 +104,21 @@ const (
 	Yes = "yes"
 	// No means no.
 	No = "no"
-
-	// AddVFSFileid is used to chec if we add vfs_fileid to the config
-	AddVFSFileid = true
 )
 
+// Constructor for the OperatorOptions struct
+func NewOperatorOptions() *OperatorOptions {
```

Comment:
This is more of a taste thing I think but I'd drop this function and simply not provide a "constructor". I'd also make it a pass-by-value (not a pointer). If you want me to elaborate on why please ask, but I'm lazy (and still on vacation ;-)) so I'm not going to elaborate yet.  However, it's minor and I could easily live with it as-is.

Reply:
Hah. Sorry, I didn't realise that you would be looking at patches today. I have just posted another one. 
Please excuse my changes as I navigate go lang. Let's schedule a 1-1 tomorrow to go through the code.